.travis.yml

```diff
@@ -3,7 +3,11 @@ rvm:
 - 2.2.2
 gemfile:
 - spec/dummy/Gemfile
-install: bundle install
+install:
+- bundle install
+- cd spec/dummy && bundle install
```

Comment:
be sure that you're working off of the latest. Rebase your changes on top of the latest.